### DISCUSSION TITLE
How is lazy.core defined?

### DISCUSSION THREAD

**Discussion by TaylorJonesTRT** [LazyVim/LazyVim#1302] [Q&A]
I am trying to take some of the functionality from LazyVim into my own config and am having a few problems figuring out where certain things get defined/loaded in.

Mainly lazyvim.util (or anything to do with lazyvim.anything like core/config/plugin).

**Comment by abeldekat** [comment]
Hello @TaylorJonesTRT,

Could you be more specific? The question on how lazy.core is defined is not clear to me. What problems do you encounter?
